frontend/src/App.vue

```diff
@@ -1,70 +1,81 @@
 <template>
-  <v-app>
+  <v-app class="bg-[#0F5892]">
     <v-main>
       <v-container
         no-gutters
-        max-width="500px"
+        max-width="800px"
+        class="text-white pa-0 mt-6 rounded-[8px] elevation-5"
+        :class="theme === 'dark' ? 'bg-[#363636]' : 'bg-[#f5f5f5]'"
       >
-        <v-select
-          v-model="selectedCameraUUID"
-          :items="cameras"
-          item-title="hostname"
-          item-value="uuid"
-          label="Camera"
-        >
-          <template #item="{ props, item }">
-            <v-list-item
-              v-bind="props"
-              :subtitle="item.raw.uuid"
-            />
-          </template>
-        </v-select>
-
-        <v-tabs
-          v-model="tab"
-          align-tabs="center"
-          class="mb-5"
-        >
-          <v-tab value="image">
-            Image
-          </v-tab>
-          <v-tab value="streams">
-            Streams
-          </v-tab>
-          <v-tab
-            value="configs"
-            :disabled="true"
-          >
-            Configs
-          </v-tab>
-        </v-tabs>
-
-        <v-tabs-window v-model="tab">
-          <v-tabs-window-item value="image">
-            <ImageTab
-              :backend-api="backendAPI"
-              :selected-camera-uuid="selectedCameraUUID"
-              :disabled="selectedCameraUUID == null"
-            />
-          </v-tabs-window-item>
-          <v-tabs-window-item value="streams">
-            <StreamsTab
-              :backend-api="backendAPI"
-              :selected-camera-uuid="selectedCameraUUID"
-              :disabled="selectedCameraUUID == null"
-            />
-          </v-tabs-window-item>
-        </v-tabs-window>
+        <div class="flex items-center justify-between bg-[#2C2C2C] rounded-t-[8px]">
+          <div class="flex items-center justify-around w-[400px] pl-5 border-b-[1px] border-[#ffffff66]">
+            <v-icon>mdi-camera-party-mode</v-icon>
+            <v-select
+              v-model="selectedCameraUUID"
+              :items="cameras"
+              item-title="hostname"
+              item-value="uuid"
+              label="Camera"
+              hide-details
+              theme="dark"
+              class="bg-[#232323] ml-3 -mb-[1px]"
+            >
+              <template #item="{ props, item }">
+                <v-list-item v-bind="props" :subtitle="item.raw.uuid" />
+              </template>
+            </v-select>
+          </div>
+          <BlueButtonGroup
+            :button-items="[
+              { name: 'Basic', onSelected: () => (configMode = 'basic'), tooltip: 'Basic setup for the Rad Cam' },
```

Comment:
I believe we'll spell it `RadCam` (as a single word). Remember to replace it anywhere else, too :)